pkg/controller/configmap/config.go

```diff
@@ -80,7 +81,7 @@ func (adaptor *ConfigMapOc) FillDefaults(configmap *corev1.ConfigMap, spec *conf
 	appendErrorIfNotNil(&errs, fillDefault(cfg, "coe", "adaptor", "openshift4", true))
 	appendErrorIfNotNil(&errs, fillDefault(cfg, "nsx_v3", "policy_nsxapi", "True", true))
 	appendErrorIfNotNil(&errs, fillDefault(cfg, "nsx_v3", "single_tier_topology", "True", true))
-	appendErrorIfNotNil(&errs, fillDefault(cfg, "nsx_v3", "wait_for_security_policy_sync", "True", false))
+	appendErrorIfNotNil(&errs, fillDefault(cfg, "nsx_v3", "wait_for_security_policy_sync", "False", true))
```

Comment:
Is it because NCP set wait_for_security_policy_sync True by default, so make it force disabled in Operator?

Reply:
I  wanted to make sure that no user can think "hey, let's try this, looks very interesting", and also - as you wrote - "fix" environments where the flag was set to True by forcing it to False.